src/Aspire.Dashboard/Components/Pages/IPageWithSessionAndUrlState.cs

```diff
@@ -55,12 +55,17 @@ public static class PageExtensions
     /// <summary>
     /// Called after a change in the view model that will affect the url associated with new page state
     /// to navigate to the new url and save new state in localstorage.
+    ///
+    /// <param name="page"></param>
+    /// <param name="layout"></param>
+    /// <param name="waitToApplyChange">Whether we should avoid applying this change immediately, and instead
```

Comment:
changed this parameter name, so it now is more descriptive of the actual behavior